apache-rat-core/src/main/java/org/apache/rat/config/exclusion/plexus/MatchPatterns.java

```diff
@@ -41,15 +42,15 @@ private MatchPatterns(final MatchPattern[] patterns) {
 
     @Override
     public String toString() {
-        return source();
+        return Arrays.stream(patterns).map(MatchPattern::toString).collect(Collectors.toList()).toString();
```

Comment:
Should we add a separator between entries to improve readability?

Reply:
Sorry misunderstanding: I did not mean reformatting of the code, but a possible separator between distinct match patterns .....

Reply:
Array#toString delivers this separator! resolved.